oci-test/tests/oci_core_subnet/variables.tf

```diff
@@ -0,0 +1,57 @@
+variable "resource_name" {
+  type        = string
+  default     = "turbot-test-20200125-create-update"
+  description = "Name of the resource used throughout the test."
+}
+
+variable "tenancy_ocid" {
+  type        = string
+  description = "OCID of your tenancy."
+}
+
+variable "config_file_profile" {
+  type        = string
+  default     = "DEFAULT"
+  description = "OCI credentials profile used for the test. Default is to use the default profile."
+}
+
+variable "oci_ad" {
+  type        = string
+  default     = "TvRS:AP-MUMBAI-1-AD-1"
+  description = "OCI region used for the test. Does not work with default region in config, so must be defined here."
+}
+
+provider "oci" {
+  tenancy_ocid = var.tenancy_ocid
+  config_file_profile = var.config_file_profile
+}
+
+resource "oci_core_vcn" "named_test_resource" {
+    compartment_id = var.tenancy_ocid
+    display_name = var.resource_name
+    cidr_block = "10.0.0.0/16"
+}
+
+resource "oci_core_subnet" "named_test_resource" {
+    compartment_id = var.tenancy_ocid
```

Comment:
Add depends on `oci_core_vcn`.

Reply:
@khushboo9024 It is necessary to add a dependency? Because we are using `vcn_id = oci_core_vcn.named_test_resource.id` inside `resource "oci_core_subnet" "named_test_resource"`.

Reply:
we add dependency to make sure that resource creation is happening in order, here we need vcn to be created first.